frontend/widgets/OBSBasic_SceneItems.cpp

```diff
@@ -35,6 +35,22 @@
 
 using namespace std;
 
+namespace {
+bool isHideInMixer(obs_source_t *source)
+{
+	OBSDataAutoRelease priv_settings = obs_source_get_private_settings(source);
+	bool hidden = obs_data_get_bool(priv_settings, "mixer_hidden");
+
+	return hidden;
+}
+
+void setHideInMixer(obs_source_t *source, bool hidden)
```

Comment:
```suggestion
void setHiddenInMixer(obs_source_t *source, bool hidden)
```
Use natural language.